src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/Indexing.scala

```diff
@@ -29,38 +43,29 @@ import scala.concurrent.Future
 private class Indexing(resources: Resources[Task], cache: Caches[Task], coordinator: ProjectViewCoordinator[Task])(
     implicit as: ActorSystem,
     config: AppConfig) {
-// TODO: uncomment when KafkaEvents are present in the AdminClient
-  private val consumerSettings = ConsumerSettings(as, new StringDeserializer, new StringDeserializer)
 
-//  private val elasticUUID = UUID.fromString("684bd815-9273-46f4-ac1c-0383d4a98254")
-//  private val sparqlUUID  = UUID.fromString("d88b71d2-b8a4-4744-bf22-2d99ef5bd26b")
+  private val logger                                          = Logger[this.type]
+  private val consumerSettings                                = ConsumerSettings(as, new StringDeserializer, new StringDeserializer)
+  private implicit val validation: AdditionalValidation[Task] = AdditionalValidation.pass
 
-//  private val defaultEsMapping =
-//    jsonContentOf("/elastic/mapping.json", Map(Pattern.quote("{{docType}}") -> config.elastic.docType))
+  private def asJson(view: View): Json =
+    view
+      .asJson(viewCtx.appendContextOf(resourceCtx))
```

Comment:
why append the context and then remove it?

Reply:
In kg we have a conversion from those models to a Graph and then to Json. That's what the `asJson` does. Now, once we have the json, I don't want to store the whole context into the primary store, just with the link to the context is enough. That's why.

Reply:
I see, it's just a code shortcut, ok.